cns/configuration/configuration.go

```diff
@@ -57,6 +57,9 @@ type CNSConfig struct {
 	UseHTTPS                    bool
 	WatchPods                   bool `json:"-"`
 	WireserverIP                string
+	EnableGRPC 									bool
+	GRPCServerIPAddress 				string
+	GRPCServerPort 							uint16
```

Comment:
To better organize the new parameters being added, let's create a struct called `GRPCSettings` to group all config settings related to gRPC. You can use something like `AZRSettings` as a reference.

Reply:
I have added a GRPCSettings struct and implemented it in CNSConfiig similar to AZSettings